recipes/paraview/5.13.2/gh200/packages.yaml

```diff
@@ -0,0 +1,11 @@
+packages:
+  all:
+    providers:
+      gl: [egl]
+  egl:
+    externals:
+    - spec: egl@1.0.0
+      prefix: /usr
+    buildable: false
```

Comment:
We have https://github.com/eth-cscs/alps-cluster-config/blob/5aa8fd7f1490462d6fc0b19880a757df00380a0d/daint/packages.yaml#L33-L37 in alps-cluster-config. Can we rely on that instead?

Reply:
can do, this was the original source of the egl settings, so it stayed here.